src/components/PageContent/Feedback.tsx

```diff
@@ -95,9 +95,9 @@ export const Feedback = () => {
         >
           <form onSubmit={handleSubmit} style={{ display: "flex", flexDirection: "column", gap: "0.3rem" }}>
             <label htmlFor="name">Tell us more about your experience.</label>
-            <textarea name="msg" />
+            <textarea name="msg" style={{ resize: "none" }} />
             <button className={button.primary} disabled={isSubmitting}>
```

Comment:
Side though: is the docs using an old version of the design system?

We now apply default styles to buttons so `className={button.primary}` is unnecessary